pkg/reaper/deployment.go

```diff
@@ -152,7 +152,7 @@ func NewDeployment(reaper *api.Reaper, dc *cassdcapi.CassandraDatacenter, keysto
 			Value: fmt.Sprintf("%d", reaper.Spec.HeapSize.Value()),
 		})
 	}
-	if reaper.Spec.HttpManagement.Enabled {
+	if reaper.Spec.HttpManagement.Enabled != nil && *reaper.Spec.HttpManagement.Enabled {
```

Comment:
[suggestion] not a hard requirement, but there is a library method for this kind of expression
```suggestion
	if pointer.BoolDeref(reaper.Spec.HttpManagement.Enabled, false) {
```